### DISCUSSION TITLE
How to lazy load images if not in viewport?

### DISCUSSION THREAD

**Discussion by masterbater** [igordanchenko/react-photo-album#118] [Q&A] [answered]
Is it possible to integrate this to images in react photo album?
https://www.npmjs.com/package/react-lazy-load-image-component

**Comment by masterbater** [comment]
I see there is a props for this
https://react-photo-album.com/documentation#RenderPhoto
But do you consider to add a lazy loading for photos not in viewport?

**Reply by igordanchenko** [reply]
Do you mean like `<img loading=“lazy”`?

**Reply by igordanchenko** [reply, marked as answer]
`react-photo-album` renders images with `loading="lazy"` attribute, so images below the fold are lazy-loaded by default.
